dts/bindings/spi/nordic,nrf-spim.yaml

```diff
@@ -25,3 +25,28 @@ properties:
         Enables the workaround for the nRF52832 SoC SPIM PAN 58 anomaly.
         Must be used in conjunction with
         CONFIG_SOC_NRF52832_ALLOW_SPIM_DESPITE_PAN_58=y
+
+    has-rx-delay:
```

Comment:
Let's use `rx-delay-supported`, for consistency with things like `dfe-supported` in the [radio binding](https://docs.zephyrproject.org/latest/build/dts/api/bindings/net/wireless/nordic%2Cnrf-radio.html#dtbinding-nordic-nrf-radio).